amgut/db/patches/0041.sql

```diff
@@ -479,6 +479,8 @@ CREATE TABLE pm.shotgun_library_prep_kit (
      water_volume_nl              real  NOT NULL,
      shotgun_i5_index_id          bigint  ,
      shotgun_i7_index_id          bigint  ,
+     qpcr_concentration           real ,
```

Comment:
real looks like it's comparable to single precision float but I didn't check if its actually ieee 754 compliant or whatever that spec is